packages/generators/templates/component/{{name}}.stories.{{mdx}}

```diff
@@ -0,0 +1,101 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { {{name}} } from ".";
+
+# {{ titleCase name }}
+
+```ts
+import { {{ name }} } from "@jobber/components/{{ name }}";
+```
+
+<Canvas>
+  <Story name="{{ name }}" args={ { "text": "Bob"} }>
+    {args => <{{ name }} {...args}" /> }
+  </Story>
+</Canvas>
+
+<!--
+  INTERFACE: This is for the proposal stage of the component
+
+  Provide an example of what the component looks like in code. How would you use
+  it from another React component? This should consist primarily of code blocks.
+  This section can be deleted once the component is built, when the Playground
+  will demonstrate this at a higher fidelity.
+-->
+
+## Props
+
+<ArgsTable of={{{ name }}} story="{{ name }}" />
```

Comment:
For some reason the `of={{{ name }}}` part didn't add `{ }` in the actual stories file 🤔 

![Screenshot 2023-02-23 at 11 45 46 AM](https://user-images.githubusercontent.com/104797704/220973598-44cb48b3-aa9d-4407-8706-b5273478327e.png)